dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/tooling/OpenTracing32.java

```diff
@@ -5,7 +5,7 @@
 import static io.opentracing.propagation.Format.Builtin.TEXT_MAP_INJECT;
 import static java.util.Collections.singletonMap;
 
-import datadog.trace.api.interceptor.MutableSpan;
+import datadog.opentracing.DDSpan;
```

Comment:
I'm going to have to take on faith that there's a plan here that won't result in lots of DD stuff. :)

Reply:
👍 we'll rename the DD stuff in #7, but the separate model (e.g. the renamed `DDSpan`) will still be there until #12